openpathsampling/experimental/simstore/tools.py

```diff
@@ -5,7 +5,7 @@
 import logging
 logger = logging.getLogger(__name__)
 
-class SimpleNamespace(collections.MutableMapping):
+class SimpleNamespace(collections.abc.MutableMapping):
```

Comment:
Any reason why this file does not do a similar thing as `memory_backend.py` (or the other way around)?
(Define `abc` and use that)

Reply:
No particular reason. I don't really have a consistent style when it comes to imports. I went ahead and switched this to just use `abc`. Really, I should have been using the [separate `abc` package for Py27](https://docs.python.org/2/library/abc.html), not `collections.MutableMapping`. I think the previous existence of that package is why I prefer `from collections import abc` (plus it is much shorter). On the other hand, I usually just do `import os` when using things in `os.path`, so there's not a general rule to apply.